packages/forma-36-react-components/src/components/Button/Button.css

```diff
@@ -240,27 +239,23 @@
 
 .Button--small,
 .Button--small:link {
-  height: calc(1rem * (30 / var(--font-base-default)));
+  height: calc(1rem * (32 / var(--font-base-default)));
```

Comment:
I wonder how the fixed height would behave with two lines button!
Should we just rely on the typography line height and padding with spacing? 
that way we get a flexible 32px, and 40px height buttons, no magic as well :)

Reply:
Do we have two lines button?

Reply:
I'm not sure. but better be flexible, this is fine for now, just sharing thoughts.